src/editor/index.js

```diff
@@ -110,14 +111,49 @@ registerPlugin( 'newspack-popups-segmentation', {
 } );
 
 registerPlugin( 'newspack-popups-dismiss', {
-	render: () => (
-		<PluginDocumentSettingPanel
-			name="popup-dismiss-panel"
-			title={ __( 'Dismiss Button Settings', 'newspack-popups' ) }
-		>
-			<DismissSidebarWithData />
-		</PluginDocumentSettingPanel>
-	),
+	render: () => {
```

Comment:
As the [linter says](https://app.circleci.com/pipelines/github/Automattic/newspack-popups/717/workflows/82089462-e673-4724-8c7b-f174a262bb70/jobs/2123/parallel-runs/0/steps/0-103), this anonymous function should be a React component.

Reply:
Is this overengineered? I made it a higher-order component in d2d7a0e.

Reply:
A little bit – since it's a single-use HOC :D But it's all right